ApollosProject/apollos-prototype
# Hides content feed item when sections are empty

### What does this PR do, or why is it needed?
There is a bug when section content is empty the titles and view all button still show.

### How do I test this PR?
1. Add var `ROCK_SHOW_INACTIVE_CONTENT=false` to api .env
2. Load simulator and click Discover tab.
3. You should only see two sections

Bug Before Fix:
https://v.usetapes.com/9oNUFIoQxT

After Fix:
https://v.usetapes.com/HqeXjkh57g

## TODO:

- [x] Closes #1153

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
